testing/MLDB-1503-bad-rowname-error-message.js

```diff
@@ -0,0 +1,51 @@
+// This file is part of MLDB. Copyright 2016 Datacratic. All rights reserved.
+
+function assertEqual(expr, val, msg)
+{
+    if (expr == val)
+        return;
+    if (JSON.stringify(expr) == JSON.stringify(val))
+        return;
+
+    throw "Assertion failure: " + msg + ": " + JSON.stringify(expr)
+        + " not equal to " + JSON.stringify(val);
+}
```

Comment:
Unrelated: Any way we can extract that? We could consider putting it into mldb js handle. Copy/pasting it everywhere clutters every test IMO.